src/plugins/discover/public/application/view_components/utils/use_search.ts

```diff
@@ -392,8 +392,7 @@ export const useSearch = (services: DiscoverViewServices) => {
       const savedSearchInstance = await getSavedSearchById(savedSearchId);
 
       const query =
-        savedSearchInstance.searchSource.getField('query') ||
-        data.query.queryString.getDefaultQuery();
```

Comment:
Since `.getDefaultQuery()` isn't being used here, is it still used in other locations? I would've assumed I would see the removal of this function as well after reading the description

Reply:
i dont know if we want removal of the function in general. but i understand what you mean it should probably be a private function within the query string manager.